bin/scripts/lib/fonts.json

```diff
@@ -615,6 +615,61 @@
       "caskName": "victor-mono",
       "repoRelease": true,
       "description": "Clean, crisp and narrow, with a large x-height and clear punctuation"
+    },
+    {
+      "unpatchedName": "MonaspaceArgon",
+      "RFN": false,
+      "patchedName": "MonaspaceArgon",
```

Comment:
I'm sorry I don't understand the subtility, we need to figure out another name such as NerdFontMonaspaceArgon ?

Reply:
Well, because of the RFN we need to rename the patched font (and thus this value here must match).
I fixed stuff in the PR directly.
Lets see how it goes with the name....